samples/browser/src/main/java/org/mozilla/samples/browser/SampleApplication.kt

```diff
@@ -58,7 +59,7 @@ class SampleApplication : Application() {
         GlobalScope.launch(Dispatchers.IO) {
             components.webAppManifestStorage.warmUpScopes(System.currentTimeMillis())
         }
-
+        components.store.dispatch(DownloadAction.RestoreDownloadsStateAction)
```

Comment:
nit: Was wondering if we should create a use case to `DownloadsUseCases` for this.

Reply:
Agree, it will hide the implementation details!